src/views/Login/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import Layout from "../../components/Layout";
+import "./Login.css";
+import * as tokenService from "../../scripts/token.js";
+
+export default class Login extends Component {
+  state = {
+    username: "",
+    password: ""
+  };
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  handleLogin = async event => {
+    event.preventDefault();
+    // await fetch("https://localhost:5001/api/token", {
+    //   method: "POST",
+    //   headers: {
+    //     Accept: "application/json",
+    //     "Content-Type": "application/json"
+    //   },
+    //   body: JSON.stringify(this.state)
+    // })
+    //   .then(function(response) {
+    //     return response.json();
+    //   })
+    //   .then(function(myJson) {
+    //     localStorage.setItem("Tokens", JSON.stringify(myJson));
+    //     console.log(JSON.stringify(myJson));
+    //   });
+    console.log("HEJ");
+    const result = await tokenService.LoginFetch({
+      method: "POST",
+      body: JSON.stringify(this.state)
+    });
```

Comment:
With the implementation from PR #19, this would be implemented like below instead.
```js
await auth.login(username, password)
```